MissileKeyboardHandler.cpp

```diff
@@ -8,6 +8,7 @@
 #include "MissileKeyboardHandler.h"
 
 MissileKeyboardHandler::MissileKeyboardHandler() {
+    // Shame this thing doesn't accept lambdas... or does it?
```

Comment:
`m_window.signal_key_press_event().connect([&](GdkEventKey *key) { return onKeyPressed(key); }, false);` fails to compile
```
$ cmake --build .                                                                                                                                             [21:32:24]
Scanning dependencies of target missile
[ 16%] Building CXX object CMakeFiles/missile.dir/usb_utils.cpp.o
[ 33%] Building CXX object CMakeFiles/missile.dir/MissileKeyboardHandler.cpp.o
In file included from /usr/include/sigc++-2.0/sigc++/signal_base.h:29:0,
                 from /usr/include/sigc++-2.0/sigc++/signal.h:8,
                 from /usr/include/sigc++-2.0/sigc++/sigc++.h:80,
                 from /usr/include/glibmm-2.4/glibmm/thread.h:58,
                 from /usr/include/glibmm-2.4/glibmm.h:87,
                 from /usr/include/gtkmm-3.0/gtkmm.h:87,
                 from /home/tsyd/src/test/missile/MissileKeyboardHandler.h:10,
                 from /home/tsyd/src/test/missile/MissileKeyboardHandler.cpp:8:
/usr/include/sigc++-2.0/sigc++/functors/slot.h: In instantiation of ‘static T_return sigc::internal::slot_call1<T_functor, T_return, T_arg1>::call_it(sigc::internal::slot_rep*, typename sigc::type_trait<T_arg3>::take) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*; typename sigc::type_trait<T_arg3>::take = _GdkEventKey* const&]’:
/usr/include/sigc++-2.0/sigc++/functors/slot.h:144:37:   required from ‘static void* (* sigc::internal::slot_call1<T_functor, T_return, T_arg1>::address())(void*) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*; sigc::internal::hook = void* (*)(void*)]’
/usr/include/sigc++-2.0/sigc++/functors/slot.h:529:91:   required from ‘sigc::slot1<T_return, T_arg1>::slot1(const T_functor&) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*]’
/usr/include/sigc++-2.0/sigc++/functors/slot.h:1157:26:   required from ‘sigc::slot<T_return, T_arg1, sigc::nil, sigc::nil, sigc::nil, sigc::nil, sigc::nil, sigc::nil>::slot(const T_functor&) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*]’
/home/tsyd/src/test/missile/MissileKeyboardHandler.cpp:11:105:   required from here
/usr/include/sigc++-2.0/sigc++/functors/slot.h:137:20: error: void value not ignored as it ought to be
                (a_1);
                    ^
In file included from /usr/include/sigc++-2.0/sigc++/functors/slot.h:7:0,
                 from /usr/include/sigc++-2.0/sigc++/signal_base.h:29,
                 from /usr/include/sigc++-2.0/sigc++/signal.h:8,
                 from /usr/include/sigc++-2.0/sigc++/sigc++.h:80,
                 from /usr/include/glibmm-2.4/glibmm/thread.h:58,
                 from /usr/include/glibmm-2.4/glibmm.h:87,
                 from /usr/include/gtkmm-3.0/gtkmm.h:87,
                 from /home/tsyd/src/test/missile/MissileKeyboardHandler.h:10,
                 from /home/tsyd/src/test/missile/MissileKeyboardHandler.cpp:8:
/usr/include/sigc++-2.0/sigc++/adaptors/adaptor_trait.h: In instantiation of ‘typename sigc::adaptor_functor<T_functor>::deduce_result_type<T_arg1>::type sigc::adaptor_functor<T_functor>::operator()(T_arg1) const [with T_arg1 = _GdkEventKey* const&; T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; typename sigc::adaptor_functor<T_functor>::deduce_result_type<T_arg1>::type = void]’:
/usr/include/sigc++-2.0/sigc++/functors/slot.h:137:20:   required from ‘static T_return sigc::internal::slot_call1<T_functor, T_return, T_arg1>::call_it(sigc::internal::slot_rep*, typename sigc::type_trait<T_arg3>::take) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*; typename sigc::type_trait<T_arg3>::take = _GdkEventKey* const&]’
/usr/include/sigc++-2.0/sigc++/functors/slot.h:144:37:   required from ‘static void* (* sigc::internal::slot_call1<T_functor, T_return, T_arg1>::address())(void*) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*; sigc::internal::hook = void* (*)(void*)]’
/usr/include/sigc++-2.0/sigc++/functors/slot.h:529:91:   required from ‘sigc::slot1<T_return, T_arg1>::slot1(const T_functor&) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*]’
/usr/include/sigc++-2.0/sigc++/functors/slot.h:1157:26:   required from ‘sigc::slot<T_return, T_arg1, sigc::nil, sigc::nil, sigc::nil, sigc::nil, sigc::nil, sigc::nil>::slot(const T_functor&) [with T_functor = MissileKeyboardHandler::MissileKeyboardHandler(Gtk::Window&)::__lambda0; T_return = bool; T_arg1 = _GdkEventKey*]’
/home/tsyd/src/test/missile/MissileKeyboardHandler.cpp:11:105:   required from here
/usr/include/sigc++-2.0/sigc++/adaptors/adaptor_trait.h:84:30: error: return-statement with a value, in function returning 'void' [-fpermissive]
     { return functor_(_A_arg1); }
                              ^
make[2]: *** [CMakeFiles/missile.dir/MissileKeyboardHandler.cpp.o] Error 1
make[1]: *** [CMakeFiles/missile.dir/all] Error 2
make: *** [all] Error 2
```